app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -43,4 +43,9 @@ object FeatureFlags {
      * Enables recording of history metadata.
      */
     val historyMetadataFeature = Config.channel.isDebug
+
+    /**
+     * Enables the recently saved bookmarks feature in the home screen.
+     */
+    val showRecentlySavedBookmarksFeature = Config.channel.isNightlyOrDebug
```

Comment:
A pet peeve that I missed when reviewing recent tabs, so barely consider this a nit 😅 :
```suggestion
    val recentBookmarksFeature = Config.channel.isNightlyOrDebug
```

The verb I think can be implied, and the shorter naming is what we're using on the feature itself.